ramlfications/loader.py

```diff
@@ -50,9 +50,14 @@ def _parse_json(self, jsonfile, base_path):
             base_path = base_path + "/"
         base_path = "file://" + base_path
 
-        with open(jsonfile, "r") as f:
-            schema = jsonref.load(f, base_uri=base_path, jsonschema=True)
-        return schema
+        try:
+            with open(jsonfile, "r") as f:
+                schema = jsonref.load(f, base_uri=base_path, jsonschema=True)
+        except ValueError as err:
+            msg = "ERROR: failed to parse JSON file: {0} ({1})"
```

Comment:
I think you're missing the `.format` to actually pass in the values to `{0}` and `{1}`.

Also - can you write a test to cover this try/except clause?